rpm/archivematica/package.spec

```diff
@@ -0,0 +1,342 @@
+cp -rf  %{_sourcedir}/%{name}/src/archivematicaCommon/lib/* %{buildroot}/usr/lib/archivematica/archivematicaCommon/
+
+mkdir -p %{buildroot}/etc/archivematica/archivematicaCommon/
+cp -rf  %{_sourcedir}/%{name}/src/archivematicaCommon/etc/* %{buildroot}/etc/archivematica/archivematicaCommon/
+
+mkdir -p %{buildroot}/var/archivematica/sharedDirectory
+cp -rf %{_sourcedir}/%{name}/src/MCPServer/share/sharedDirectoryStructure/* %{buildroot}/var/archivematica/sharedDirectory/
+
+
+%prep
+rm -rf /usr/share/archivematica
+rm -rf /usr/lib/python2.7/archivematica
+rm -rf %{_sourcedir}/*
+rm -rf %{buildroot}/*
+mkdir -p %{buildroot}/%{install_dir}
+
+git clone --branch stable/1.4.x --depth 1 --single-branch https://github.com/artefactual/archivematica %{_sourcedir}/%{name}
+
+cd %{_sourcedir}/%{name} && git submodule init && git submodule update
+
+%clean
+rm -rf %{buildroot}
+
+
+# Post install scripts
+%post
+# Dashboard
+echo "Creating archivematica user"
+userID=`id -u archivematica`
+
+if [ "${userID}" = 333 ]; then
+  echo "User archivematica exists"
+else
+  useradd --uid 333 --user-group --home /var/lib/archivematica/ archivematica
+fi
+
+
+echo "Creating archivematicadashboard user"
+userID=`id -u archivematicadashboard`
+
+if [ "${userID}" = 334 ]; then
+  echo "User archivematicadashboard exists"
+else
+  useradd --uid 334 --gid archivematica --home /var/lib/archivematicadashboard/ archivematicadashboard
+fi
+
+echo "Update SELinux policies"
+if [ x$(semanage port -l | grep http_port_t | grep 7600 | wc -l) == x0 ]; then
+  semanage port -a -t http_port_t -p tcp 7600
+fi
+
+
+echo "Creating log directories"
+mkdir -p /var/log/archivematica/dashboard/
+touch /var/log/archivematica/dashboard/dashboard.log
+touch /var/log/archivematica/dashboard/dashboard_debug.log
+chown -R archivematicadashboard:archivematica /var/log/archivematica/dashboard
+chmod 660 /var/log/archivematica/dashboard/*
+
+# MCP Server
+mkdir -p /var/log/archivematica/MCPServer/
+touch /var/log/archivematica/MCPServer/MCPServer.log
+touch /var/log/archivematica/MCPServer/MCPServer_debug.log
+chown -R archivematica:archivematica /var/log/archivematica/MCPServer
+
+# MCP Client
+%post mcp-client
+echo "creating archivematica user"
+userID=`id -u archivematica`
+
+if [ "${userID}" = 333 ]; then
+  echo "User archivematica exists"
+else
+  useradd --uid 333 -U --home /var/lib/archivematica/ archivematica
+fi
+
+logdir=/var/log/archivematica/MCPClient
+sudo mkdir -p $logdir
+sudo chown -R archivematica:archivematica $logdir
+sudo chmod -R g+ws $logdir
+
+# Common
+%post common
+echo "Creating archivematica user"
+userID=`id -u archivematica`
+
+if [ "${userID}" = 333 ]; then
+  echo "User archivematica exists"
+else
+  useradd --uid 333 --user-group --home /var/lib/archivematica/ archivematica
+fi
+
+chown -R archivematica:archivematica /var/archivematica/sharedDirectory
+chmod 770 -R /var/archivematica/sharedDirectory
+
+/usr/share/archivematica/postinstSharedWithDev
```

Comment:
The sudo permissions are managed with this script, so I added it, but we can add the configuration steps to the rpm and avoid using it.